gepetuto/main.py

```diff
@@ -97,26 +105,32 @@ def retrieve_python_interpreter():
             return sys.executable
 
 
-def get_file_list(tp_id, file):
+def get_file_list(tp_id, file, filters):
     """Get the list of files we use action on."""
     file = [Path(f) for f in file]
     file_list = []
     if tp_id == []:
         tp_id = get_tp_id()
     for n in tp_id:
         folder = Path(f"tp{n}")
-        if file == []:
-            file_list += folder.glob("*.py")
-        else:
-            file_list += list(filter(lambda f: f in file, folder.glob("*.py")))
+        tp_files = folder.glob("*.py")
+        if file != []:
+            tp_files = list(filter(lambda f: f in file, tp_files))
```

Comment:
```suggestion
            tp_files = [f for f in tp_files if f in file]
```
comprehension lists are more idiomatic in python than then functional approach